verification/infrastructure/models.py

```diff
@@ -42,3 +42,14 @@ class DUNSVerificationModel(Base):
     status = Column("status", VARCHAR(255))
     created_at = Column("created_at", TIMESTAMP(timezone=False))
     updated_at = Column("updated_at", TIMESTAMP(timezone=False))
+
+
+class IndividualVerificationModel(Base):
+    __tablename__ = "individual_verification"
+    row_id = Column("row_id", Integer, autoincrement=True, primary_key=True)
```

Comment:
We should have Unique constraint. By default nullable is True, verification_id  and username can't be None.

Reply:
https://github.com/singnet/snet-marketplace-service/pull/1431